.rat-excludes

```diff
@@ -83,3 +83,5 @@ gen-java.*
 .*avpr
 org.apache.spark.sql.sources.DataSourceRegister
 .*parquet
+spark-deps-hadoop1
```

Comment:
Nit: does this work if they're in `dev/`, or if it does should it be prefixed to be clear what this is excluding? I'm typing off the top of my head so you may tell me I'm just forgetting how rat excludes work.